### PR TITLE
FeddTag: add func for fetching next feedTag page

### PR DESCRIPTION
**Pull request by sviande**
Hi this PR : add func Next on feedTag for fetching next pages (Like Hastag.Next)

Usage:
```
ft, err := inst.Feed.Tags("mytag")
if err != nil {
   return err
}
dosomethingWithItems(ft)
for ft.Next() {
  dosomethingWithItems(ft)
}

return ft.Error()
```

### CODE REVIEW

File: feeds.go [hunk 1]
```diff
@@ -101,3 +96,59 @@ type FeedTag struct {
...
+				"rank_token": insta.rankToken,
+			},
+			Endpoint: fmt.Sprintf(urlFeedTag, name),
+		},
+	)
+	if err == nil {
+		newFT := FeedTag{}
+		err = json.Unmarshal(body, &newFT)
+		if err == nil {
+			*ft = newFT
```
Comment (ahmdrz) [inline feeds.go]: Are these lines the same as yours?
```go
		newFT := &FeedTag{}
		err = json.Unmarshal(body, newFT)
		if err == nil {
			ft = newFT
```

File: feeds.go [hunk 2]
```diff
@@ -73,25 +73,20 @@ func (feed *Feed) Tags(tag string) (*FeedTag, error) {
...
 
 	return res, nil
 }
 
 // FeedTag is the struct that fits the structure returned by instagram on TagSearch.
 type FeedTag struct {
+	inst *Instagram
+	err  error
+
+	Name string
```
Comment (ahmdrz) [inline feeds.go]: You can make it private.

File: feeds.go (hunk 1 above)
Comment (sviande) [reply feeds.go]: Yes it's the same, i copy the code from hashtags.go

Comment (ahmdrz) [reply feeds.go]: Alright, Later I will change both of them.

**Comment by ahmdrz** [comment]
Thank you @sviande. Can you create some test functions for feed tags? I'll merge it BTW.

**Comment by sviande** [comment]
>  Can you create some test functions for feed tags? I'll merge it BTW.
I will add some tests and create a new PR.

**Pull request merged**